pkg/apiserver/metrics/metrics.go

```diff
@@ -42,7 +42,7 @@ var (
 			// Use buckets ranging from 125 ms to 8 seconds.
 			Buckets: prometheus.ExponentialBuckets(125000, 2.0, 7),
 		},
-		[]string{"verb", "resource"},
+		[]string{"verb", "resource", "client"},
```

Comment:
I'm not terribly concerned about how much memory, as we load-balance at higher levels, and there are caps on the number of connections. 

I guess a better question is, what is your objective in tooling per-client.